stdlib/Test/src/Test.jl

```diff
@@ -1101,7 +1105,11 @@ struct FailFastError <: Exception end
 # For a broken result, simply store the result
 record(ts::DefaultTestSet, t::Broken) = (push!(ts.results, t); t)
 # For a passed result, do not store the result since it uses a lot of memory
```

Comment:
```suggestion
```

This comment should be removed/changed to mention that this only happens conditionally with this PR.

Reply:
Thanks. Yeah this is more of an investigation of the performance/test side effect implications currently.